### PR TITLE
🌐 Fix typo in Russian translation for `docs/ru/docs/tutorial/body-fields.md`

### PR DESCRIPTION
**Pull request by AlertRED**

### CODE REVIEW

**Review by Kludex — APPROVED** [review]
Approving to make it easier for @tiangolo to clean up PRs. I do not speak Russian, nor I've used the translator to verify this.

**Comment by tiangolo** [comment]
📝 Docs preview for commit c2c92fd8483b21e714a6476d9e318005e54e667b at: https://f869b00e.fastapitiangolo.pages.dev

**Comment by alex-pobeditel-2004** [comment]
*Giving a russian-native approve* :)  
Proper fix.

**Review by AGolicyn — APPROVED** [review]

File: docs/ru/docs/tutorial/body-fields.md [hunk 1]
```diff
@@ -37,7 +37,7 @@
     {!> ../../../docs_src/body_fields/tutorial001.py!}
     ```
 
-Функция `Field` работает так же, как `Query`, `Path` и `Body`, у ее такие же параметры и т.д.
+Функция `Field` работает так же, как `Query`, `Path` и `Body`, у нее такие же параметры и т.д.
```
Comment (Xewus) [inline docs/ru/docs/tutorial/body-fields.md:40]:
```suggestion
Функция `Field` работает так же, как `Query`, `Path` и `Body`, у неё такие же параметры и т.д.
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7e69a81748f108122a0da7ec37ea9bb6c66bf936 at: https://09dbf711.fastapitiangolo.pages.dev

**Review by Xewus — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @AlertRED! 🍰 

And thanks for the reviews @Xewus, @AGolicyn! ☕

**Pull request merged**